### PR TITLE
Rerun through DG and update them

### PR DESCRIPTION
**Pull request by aryagiat**
Update DG and UG that is outdated

### CODE REVIEW

**Comment by ryanpeh** [comment]
There's some checkstyle errors preventing us from merging this in.

File: docs/tutorials/AddRemark.md [hunk 1]
```diff
@@ -284,29 +277,24 @@ Unfortunately, a change to `Question` will cause other commands to break, you wi
...
 
-AddressBook stores data by serializing `JsonAdaptedQuestion` into `json` with the help of an external library — Jackson. Let’s update `JsonAdaptedQuestion` to work with our new `Question`!
+SmartNus stores data by serializing `JsonAdaptedQuestion` into `json` with the help of an external library — Jackson. Let’s update `JsonAdaptedQuestion` to work with our new `Question`!
 
 While the changes to code may be minimal, the test data will have to be updated as well.
 
 <div markdown="span" class="alert alert-warning">
 
-:exclamation: You must delete AddressBook’s storage file located at `/data/addressbook.json` before running it! Not doing so will cause AddressBook to default to an empty address book!
+:exclamation: You must delete SmartNus' storage file located at `/data/smartnus.json` before running it! Not doing so will cause SmartNus to default to an empty questin list!
```
Comment (Agentum07) [inline docs/tutorials/AddRemark.md:289]: Typo.
Correct line: 
:exclamation: You must delete SmartNus' storage file located at `/data/smartnus.json` before running it! Not doing so will cause SmartNus to default to an empty question list!

**Review by Agentum07 — DISMISSED** [review]
lgtm!

**Review by Agentum07 — APPROVED** [review]
lgtm!

**Pull request merged**